### PR TITLE
add fb banner

### PR DESCRIPTION
**Pull request by codiebeulaine**

### CODE REVIEW

**Comment by codiebeulaine** [comment]
screen shot 2017-12-08 at 2 58 29 pm

File: gem/settings/base.py [hunk 1]
```diff
@@ -133,7 +133,7 @@
 
 # We have multiple layouts: use `base`, `malawi` or `springster`
 # to switch between them.
-SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'base')
+SITE_LAYOUT_BASE = environ.get('SITE_LAYOUT_BASE', 'springster')
```
Comment (saeedmb8) [inline gem/settings/base.py]: @codiebeulaine please change this back to base

File: gem/migrations/0022_auto_20171208_1336.py [hunk 2]
```diff
@@ -0,0 +1,33 @@
+# -*- coding: utf-8 -*-
```
Comment (saeedmb8) [inline gem/migrations/0022_auto_20171208_1336.py]: can you please give the migration a name

File: gem/templatetags/gem_tags.py [hunk 3]
```diff
@@ -14,6 +16,31 @@ def fieldtype(field):
...
+
+    if request.site:
+        pages = request.site.root_page.specific.bannerpages().exact_type(
+            GemTextBanner)
+    else:
+        pages = []
+    return {
+        'bannerpages': get_pages(context, pages, locale),
+        'request': context['request'],
+        'locale_code': locale,
```
Comment (saeedmb8) [inline gem/templatetags/gem_tags.py]: Suggestion: if we are going to have a template tag for the banner you could just pass in the `is_via_freebasics: context['is_via_freebasics']` and not have the `is_via_freebasics` template tag. but happy to have the template tag if you prefer it that way

**Comment by saeedmb8** [comment]
left a suggestion there but if you happy with the current approach then 👍 🍰

**Pull request merged**